src/main/java/com/johnstarich/moviematcher/models/Rating.java

```diff
@@ -42,4 +43,9 @@ public boolean equals(Object o) {
     public static List<Rating> loadRatingsByUser(ObjectId userID) {
         return MovieMatcherDatabase.morphium.findByField(Rating.class, "user_id", userID);
     }
+
+    public static Optional<Rating> loadRatingByUser(ObjectId userId, ObjectId movieId) {
+        List<Rating> ratings = loadRatingsByUser(userId);
```

Comment:
This will potentially cause problems in the future since loading all ratings by a user into the API's memory would take a long time and a lot of space in memory if they rated many movies.

The simple solution would be to just put this filter statement into the query itself